examples/b2c/checkout/integrations/classes/B2CDeliverySample.cls

```diff
@@ -258,7 +259,7 @@ global class B2CDeliverySample implements sfdc_checkout.CartShippingCharges {
         for (ShippingOptionsAndRatesFromExternalService shippingOption: shippingOptions) {
             String shippingOptionNumber = String.valueOf(i);
             String name = defaultDeliveryMethodName + shippingOptionNumber;
-            List<OrderDeliveryMethod> odms = [SELECT Id, ProductId, Carrier, ClassOfService FROM OrderDeliveryMethod WHERE Name = :name];
+            List<OrderDeliveryMethod> odms = [SELECT Id, ProductId, Carrier, ClassOfService FROM OrderDeliveryMethod WHERE Name = :name WITH SECURITY_ENFORCED];
```

Comment:
Do we want to introduce `WITH SECURITY_ENFORCED`? I did not see that in Aditya's PR

Reply:
These are his changes - https://github.com/forcedotcom/commerce-on-lightning/pull/214/files

Reply:
It looks like with this, there are no ODMs created